openkfw/Oscar
# 69 update geodata structure

## Description
MapLayer table is updated and is now storing geoMetadata in metadata property and returns it on frontend and its also returning format. You should see metadata on frontend, when you click on info icon. 
this branch is parent for: https://github.com/openkfw/Oscar/pull/84


## Motivation and Context




## How has this been tested?




- run ./start.sh, ./runinitialload.sh, with Sample country in .env
- format should be stored in layerGeoData collection
- geoMeatadata should be stored in metadata in mapLayers collection
- check staticLayers on frontend, there should be also geoMetadata in metadata and format returned 
- check documentation, there are some parts where geojson is meant as only format to use in app, but I will refactor this in the next task where gpkg format will be used.

## Screenshots (if appropriate):

## Types of changes




## Checklist:
